sql-plugin/src/main/320+/scala/org/apache/spark/sql/rapids/shims/AvroUtils.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.rapids.shims
+
+import com.nvidia.spark.rapids.RapidsMeta
+
+import org.apache.spark.sql.avro.AvroOptions
+
+object AvroUtils {
+
+  def tagSupport(
+    parsedOptions: AvroOptions,
+    meta: RapidsMeta[_, _, _]): Unit = {
+
+    if (parsedOptions.positionalFieldMatching) {
+      meta.willNotWorkOnGpu("GpuAvroScan does not support positionalFieldMatching")
```

Comment:
This meta is already associated with Avro, so `GpuAvroScan` is redundant.  Also this is user-facing, so we don't want to use parameter names from code here if the user won't be familiar with them.
```suggestion
      meta.willNotWorkOnGpu("positional field matching is not supported")
```

Reply:
Done